src/client/cli/formatter/yaml_formatter.cpp

```diff
@@ -66,7 +66,7 @@ std::string mp::YamlFormatter::format(const InfoReply& reply) const
             std::locale::global(std::locale("C"));
             auto loads = mp::utils::split(info.load(), " ");
             for (const auto& entry : loads)
-                instance_node["load"].push_back(std::stod(entry));
+                instance_node["load"].push_back(entry);
```

Comment:
This didn't work with new yaml-cpp, or did you just tweak it to look better?

Reply:
The new `yaml-cpp` switched to a new precision type, so we'd end up with long decimal values if `load` was kind of high.  When I saw this, I thought to myself that we should leave it as a string just as the `load` outputs and not modify it.

Reply:
ACKies